doc/neps/nep-0000.rst

```diff
@@ -1,391 +1,172 @@
-   discussions of API details.
+Maintenance
+^^^^^^^^^^^
 
-   The final implementation must include test code and documentation
-   appropriate for either the Python language reference or the
-   standard library reference.
+In general, Standards track NEPs are no longer modified after they have
+reached the Final state.
+However, finalized Standards track NEPs may be updated as needed.
 
+Informational and Process NEPs may be updated over time to reflect changes
+to development practices and other details. The precise process followed in
+these cases will depend on the nature and purpose of the NEP being updated.
 
-PEP Formats and Templates
-=========================
 
-PEPs are UTF-8 encoded text files using the reStructuredText_ format.
-ReStructuredText_ allows for rich markup that is still quite easy to
-read, but also results in good-looking and functional HTML. PEP 12
-contains instructions and a template [4]_ for reStructuredText PEPs.
+Format and Template
+-------------------
 
-A Python script automatically converts PEPs to HTML for viewing on
-the web [5]_.  The conversion of reStructuredText PEPs is handled by
-the Docutils_ module; the same script also renders a legacy plain-text
-format of PEP internally, to support pre-reST documents.
+NEPs are UTF-8 encoded text files using the reStructuredText_ format.  Please
+see the :doc:`nep-template` file and the reStructuredTextPrimer_ for more
+information.  We use Sphinx_ to convert NEPs to HTML for viewing on the web
+[2]_.
 
 
-PEP Header Preamble
-===================
+Header Preamble
+^^^^^^^^^^^^^^^
 
-Each PEP must begin with an RFC 822 style header preamble.  The headers
-must appear in the following order.  Headers marked with "*" are
-optional and are described below.  All other headers are required. ::
+Each NEP must begin with a header preamble.  The headers
+must appear in the following order.  Headers marked with ``*`` are
+optional.  All other headers are required. ::
 
-    PEP: <pep number>
-    Title: <pep title>
-    Author: <list of authors' real names and optionally, email addrs>
-  * BDFL-Delegate: <PEP czar's real name>
-  * Discussions-To: <email address>
-    Status: <Draft | Active | Accepted | Deferred | Rejected |
+    :Author: <list of authors' real names and optionally, email addresses>
+    :Status: <Draft | Active | Accepted | Deferred | Rejected |
              Withdrawn | Final | Superseded>
-    Type: <Standards Track | Informational | Process>
-  * Content-Type: <text/x-rst | text/plain>
-  * Requires: <pep numbers>
-    Created: <date created on, in dd-mmm-yyyy format>
-  * Python-Version: <version number>
-    Post-History: <dates of postings to python-ideas and/or python-dev>
-  * Replaces: <pep number>
-  * Superseded-By: <pep number>
-  * Resolution: <url>
+    :Type: <Standards Track | Informational | Process>
+    :Created: <date created on, in dd-mmm-yyyy format>
+  * :Requires: <nep numbers>
+  * :Python-Version: <version number>
```

Comment:
Hey, managed to find one leftover! Do we need both python and numpy versions?

Reply:
Thanks!

Reply:
Fixed and rebased on master.

Reply:
Changed
```
:Python-Version: <version number>
```
to
```
:NumPy-Version: <version number>
```